Framework/DataHandling/test/SaveGSASInstrumentFileTest.h

```diff
@@ -190,61 +190,61 @@ class SaveGSASInstrumentFileTest : public CxxTest::TestSuite {
     vector<string> parnames;
     vector<double> parvalues;
 
-    parnames.push_back("BANK");
+    parnames.emplace_back("BANK");
```

Comment:
I didn't change this to an initializer list since one would no longer have values in parnames and parvalues with the same index next to each other.